app/models/manual.rb

```diff
@@ -0,0 +1,163 @@
+  end
+
+  %w{draft live redrafted}.each do |state|
+    define_method("#{state}?") do
+      publication_state == state
+    end
+  end
+
+  def updated_at
+    @updated_at ||= Time.zone.now
+  end
+
+  def updated_at=(timestamp)
+    @updated_at = Time.parse(timestamp.to_s) unless timestamp.nil?
+  end
+
+  def public_updated_at
+    @public_updated_at ||= Time.zone.now
+  end
+
+  def public_updated_at=(timestamp)
+    @public_updated_at = Time.parse(timestamp.to_s) unless timestamp.nil?
+  end
+
+  def sections=(section_content_ids)
+    @section_content_ids = section_content_ids
+  end
+
+  def sections
+    @sections ||= @section_content_ids.map { |content_id|
+      Section.find(content_id: content_id)
+    }
+  end
+
+  def organisations=(organisation_content_ids)
+    @organisation_content_ids = organisation_content_ids
+  end
+
+  OrganisationStruct = Struct.new(:content_id, :base_path, :title)
+
+  def organisations
+    @organisations ||= @organisation_content_ids.map { |content_id|
+      payload = publishing_api.get_content(content_id).to_hash
+      OrganisationStruct.new(payload["content_id"], payload["base_path"], payload["title"])
+    }
+  end
+
+  def self.all
+    # Fetch individual payloads and links for each `manual`
+    payloads = self.content_ids.map { |payload|
+      publishing_api.get_content(payload.content_id).to_hash.deep_merge!(
+        publishing_api.get_links(payload.content_id).to_hash
+      )
+    }
+
+    # Deserialize the payloads into real Objects and return them
+    payloads.map { |payload| self.from_publishing_api(payload) }
+  end
+
+  def self.where(organisation_content_id:)
+    # Fetch individual links for each `manual`
+    payloads = self.content_ids.map { |payload|
+      publishing_api.get_links(payload.content_id).to_ostruct
+    }
+
+    # Select ones which have the same `content_id` as the `organisation_content_id` arguement
+    payloads.select! { |payload| payload.links.organisations.present? }
+    payloads.select! { |payload| payload.links.organisations.include?(organisation_content_id) }
+
+    # Fetch the content_id
+    payloads = payloads.map { |payload|
+      content = publishing_api.get_content(payload.content_id).to_hash
+      content.deep_merge!(payload.links)
+    }
+
+    # Deserialize the payloads into real Objects and return them
+    payloads.map { |payload| self.from_publishing_api(payload) }
+  end
+
+  def self.find(content_id:, organisation_content_id: nil)
+    if organisation_content_id
+      links = publishing_api.get_links(content_id).o_struct
+
+      if links.organisations.include?(organisation_content_id)
+        content = publishing_api.get_links(content_id).to_hash
+        payload = content.deep_merge(links.to_hash)
+      else
+        return nil
+      end
+    else
+      content = publishing_api.get_content(content_id).to_hash
+      links = publishing_api.get_links(content_id).to_hash
+      payload = content.deep_merge(links)
+    end
```

Comment:
We can do, but I'm not sure if it's going to be that common. The reason I'm having to do this here is in order to filter by `organisation`. When we have a proper search interface, I won't need to do this in within the App.